contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,101 @@
+
+At a lot of companies, this has forced way too many people onto the “manager’s schedule.” If this is your company, burn your weekly schedule to the ground and start again. At PostHog, we’re [anti-meeting by default](https://posthog.com/blog/meetings). We have two meeting-free days a week and, when we do have meetings, they’re short and focused. This allows not just engineers, but everyone at the company, to focus on doing their best work how they see best. We communicate asynchronously via GitHub and Slack. 
+
+A lot of companies, big and small, have adopted this approach. GitLab and Automattic for starters, but also [Twist](https://async.twist.com/asynchronous-communication/), [Buffer](https://buffer.com/about), and [Zapier](https://zapier.com/jobs/culture-and-values-at-zapier) have all followed a similar blueprint to make remote work actually work for both managers and engineers.
+
+### The problem: Everyone will slack off
+
+This is probably the biggest (and stupidest) argument against remote working. Statistics on this issue abound, but here are some key findings: remote work [doesn’t affect productivity, at all](https://www.bloomberg.com/news/articles/2024-01-16/remote-work-doesn-t-seem-to-affect-productivity-fed-study-finds). That’s not according to some random survey, that’s based on detailed research by the Federal Reserve Bank of San Francisco. 
+
+Study after study has also shown that remote workers are happier, more diverse, and that they stay at companies for longer and, as a result, do better work. That’s especially true [for engineers and developers](https://www.shakebugs.com/blog/why-let-developers-work-remotely/). Stackoverflow has a great guide that navigates the confusing [mess of statistics](https://stackoverflow.blog/2023/11/27/are-remote-workers-more-productive-that-s-the-wrong-question/) around productivity and remote work.
+
+Ultimately, companies that want to make remote working work – and create a motivated, ambitious, and productive working environment – have to put a huge amount of trust in their employees. So, perhaps counter-intuitively, the solution to the perceived problem of productivity amongst remote workers is to go all-in on trust.
+
+### The solution: Trust like you’ve never trusted before
+
+What does that mean in practice? First up, employers need to set clear and transparent expectations and then, within that framework, give people total freedom to get the job done. This kind of framework is great for engineers TKTK. Buffer co-founder and CEO Joel Gascoigne laid all this out [almost a decade ago](https://buffer.com/resources/why-transparency/). It’s still true today.
```

Comment:
As per earlier comments, I'd like to see more examples of how real companies put this into practice + some examples of benefits if they're convincing. Just build the whole section around examples, rather than talking generally about the benefits of doing it.

Also "First up, employers need to set clear and transparent expectations and then, within that framework, give people total freedom to get the job done." just read like management speak. In general, we try to avoid the word "framework" where possible. I think it's a red flag for engineers.

Reply:
```suggestion
What does that mean in practice? First up, employers need to set clear and transparent expectations and then, within that framework, give people total freedom to get the job done. Buffer co-founder and CEO Joel Gascoigne laid all this out [almost a decade ago](https://buffer.com/resources/why-transparency/). It’s still true today.
```

Reply:
Whoops, that was a placeholder. And if you've got any examples of some other companies/things companies are doing that would be a good fit here that would be really useful!